### PR TITLE
Identify both mimetype and filetype.

### LINKED ISSUE #30: Store mimetype and pronom id
Per guidance from @andrewjbtw. 

Currently, only storing mimetype.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #30

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -21,10 +21,6 @@ Metrics/BlockLength:
   Exclude:
     - 'spec/**/*'
 
-Metrics/BlockLength:
-  Exclude:
-    - spec/**/*.rb
```
Comment (jcoyne) [inline .rubocop.yml]: why did this get removed? It's quite usual for rspec blocks to get longer than 5 lines.

File: app/services/file_identifier_service.rb [hunk 2]
```diff
@@ -33,18 +33,28 @@ def version
...
       end
     end
 
     raise Error, "Unable to find file type for #{filepath} in: #{output}"
   end
+
+  def extract_pronom_id(match)
+    return nil if match['puid'] == 'UNKNOWN'
+
+    match['puid'].presence
```
Comment (jcoyne) [inline app/services/file_identifier_service.rb]: why `#presence` here?

File: spec/services/file_identifier_service_spec.rb [hunk 3]
```diff
@@ -56,8 +56,8 @@
 
       let(:status) { instance_double(Process::Status, success?: true) }
 
-      it 'returns mimetype' do
-        expect(mimetype).to eq('text/plain')
+      it 'returns pronom id and mimetype' do
+        expect(mimetype).to eq(['x-fmt/111', 'text/plain'])
```
Comment (jcoyne) [inline spec/services/file_identifier_service_spec.rb]: how about changing the name of the subject from `mimetype` to something more descriptive now?

File: .rubocop.yml (hunk 1 above)
Comment (justinlittman) [reply .rubocop.yml]: Repeat of rule above.

File: app/services/file_identifier_service.rb (hunk 2 above)
Comment (justinlittman) [reply app/services/file_identifier_service.rb]: So that `''` returns nil.

File: spec/services/file_identifier_service_spec.rb (hunk 3 above)
Comment (justinlittman) [reply spec/services/file_identifier_service_spec.rb]: Done.

**Pull request merged**